gatsby/src/pages/404.tsx

```diff
@@ -1,12 +1,39 @@
 import React from "react";
 import { Layout } from "../components/Layout/Layout";
 import { SEO } from "../components/SEO/SEO";
+import { Link } from "gatsby";
```

Comment:
Nitpicking but typically we try to orgainise imports so 3rd party deps are first, then NICE imports from npm then relative imports. Don't have a linting rule for this so it's more convention, and like I say, proper nitpicking so don't worry too much about it!